test/Microsoft.ML.FSharp.Tests/SmokeTests.fs

```diff
@@ -74,7 +74,7 @@ module SmokeTest1 =
         [<ColumnName("PredictedLabel"); DefaultValue>]
         val mutable Sentiment : bool
 
-    [<Fact>]
+    [<Fact(Skip = "Needs to be updated to not use Legacy API.")>]
```

Comment:
I don't think this is a valid approach. We can't shut off tests, because then the next change will just break these tests further.

We need to keep our tests running - it is what they are there for.

Reply:
Forced eager loading of the assembly here to fix the test.

---
In reply to: [240801842](https://github.com/dotnet/machinelearning/pull/1859#discussion_r240801842) [](ancestors = 240801842)